src/local/butler/scripts/migration/monorail_to_issuetracker_migrator.py

```diff
@@ -43,15 +44,15 @@ def execute(args):
 
     testcases.append(testcase)
 
-    if args.non_dry_run and len(testcases) > 499:  # maximum batch size is 500
+    if args.non_dry_run and len(testcases) > args.batch_size:
       ndb.put_multi(testcases)
       testcases = []
 
   if args.non_dry_run and len(testcases) > 0:
     ndb.put_multi(testcases)
 
```

Comment:
Can we add logging?

Reply:
Done. I suspect once we can run it locally we will have to tweak this logging to see what is useful to see.